frog/imports/ui/GraphEditor/fixedComponents.js

```diff
@@ -58,21 +58,29 @@ export const LevelLines = connect(({
 
 export const TimeScale = connect(({
   store: { ui: { scale }, graphDuration }
-}) => (
-  <g>
-    {[...Array(graphDuration).keys()].map(index => {
-      const i = index + 1;
-      const boolToBit = b => b ? 1 : 0;
-      const length = boolToBit(i % 15 === 0) * 15 +
-        boolToBit(i % 5 === 0) * 10 +
-        5;
-      const x = timeToPx(i, scale);
-      return (
-        <g key={i}>
-          <line x1={x} y1={600 - length} x2={x} y2={600} stroke="grey" />
-          {i % 15 === 0 ? <text x={x - 15} y={540}>{i + ' min.'}</text> : null}
-        </g>
-      );
-    })}
-  </g>
-));
+}) => {
+  let divider = Math.round(5 / scale) % 5 * 5;
+  if (divider === 0) {
+    divider = 20;
+  }
```

Comment:
The `% 5` followed by handling the case of `0` can be made a lot more clear with the `between` method. Example of code that does the same thing: 
```
const divider = Math.round(between(1, 4, 5 / scale)) * 5
```